assets/src/realtime.ts

```diff
@@ -7,7 +7,7 @@ import {
   TimepointId,
   TripId,
   ViaVariant,
-} from "./schedule.d"
+} from "./schedule"
```

Comment:
Curiosity: Is there another `schedule.d.ts` ==> `schedule.ts` change associated with this, or can you import things from `*.d.ts` without the `.d`?

Looks like the build is passing, so it looks like this didn't break anything O_o

Reply:
AFAIK you can leave off all the extensions when importing, didn't notice this change though sorry!

I'd be open to reverting that if you feel strongly enough about it.

Reply:
No feelings whatsoever - I was just surprised that it worked.